src/main/java/roomescape/domain/Reservation.java

```diff
@@ -0,0 +1,36 @@
+package roomescape.domain;
+
+public class Reservation {
+    private int id;
+
+    private String name;
+    private String date;
+    private Time time;
```

Comment:
> 서비스 계층에서 다른 서비스를 의존성 주입받아도 되는 지...?

```suggestion
public class Reservation {
    private int id;

    private String name;
    private String date;
    private Long timeId;
```

- 이런식으로 id로 매핑한다면 Time 객체를 얻기 위한 코드도 불필요해지니 의존성 문제를 해결할 수 있지 않을까요?